src/util/jszip.js

```diff
@@ -1,9155 +1,9412 @@
 /*!
```

Comment:
I went back and forth with them for so long that I had forgotten that I wasn't supposed to update them, as I did when I had the module resolution conflict to see if that solved it. That update may still be worthwhile but it's outside the scope of this pull request. I'll put the previous versions and add the pure AMD interface to proj4-src.